crates/relayer/src/chain/cosmos.rs

```diff
@@ -1893,9 +1910,17 @@ fn do_health_check(chain: &CosmosSdkChain) -> Result<(), Error> {
         );
     }
 
+    // Check for `min_gas_price` here
+    // ibc_proto::cosmos::staking::v1beta1::query_client::QueryClient::connect(
```

Comment:
Just a note that ICS consumer chains don't have a staking module and that this query may this fail. I'll check if we can get this information some other way for those chains but in any case we should gracefully handle the case where the query fails because the module is not enabled.

Reply:
We also need to do the same for the unbonding period.

Reply:
> we should gracefully handle the case where the query fails because the module is not enabled

Does this implementation of `query_config_params` handle this? I'm not very clear on that.